include/mgos_sntp.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+double mgos_sntp_get_last_synced_uptime(void);
+
+bool mgos_sntp_is_synced(void);
```

Comment:
i think we don't need this method, just add comment to mgos_sntp_get_last_synced_uptime that it returns 0 until synced.

Reply:
It's just a shorthand for the mos frequent use of this API, including https://github.com/mongoose-os-libs/sntp/blob/f7f4c04b8ce1231d6fa3fda3e70a8f5a11e28220/src/mgos_sntp.c#L115. But OK.